canonical/ubuntu.com
# Update the Ubuntu Pro service description

## Done

- Update the link to the Ubuntu Pro service description to the new PDF 

## QA

- Open the demo
- Go to /legal/ubuntu-pro
- Click the service description link and see it opens the PDF
